### PR TITLE
Enable editor unit for usage

### PR DESCRIPTION
**Pull request by korvin89**
Enable `ChartEditor` unit in development env

### CODE REVIEW

File: src/server/components/features/features-list/EnableChartEditor.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {createFeatureConfig} from '../utils';
 export default createFeatureConfig({
     name: Feature.EnableChartEditor,
     state: {
-        development: false,
+        development: true,
```
Comment (melikhov-dev) [inline src/server/components/features/features-list/EnableChartEditor.ts:7]: This flag can enable `save as editor script` button for wizard. Do you really need them?

**Comment by Marginy605** [comment]
Let's change default example for editor selector to this one:
```
module.exports = [
    {
        type: 'select' ,
        param: 'scale',
        content: [
            {title: 'Daily', value: 'd'},
            {title: 'Weekly', value: 'w'},
            {title: 'Monthly', value: 'm'}
        ],

        // updateOnChange: true,
        // updateControlsOnChange: true
    },
    {
        type: 'input',
        param: 'input',
        
        innerLabel: 'Search',
        
        // Placeholder text, when nothing is selected
        // Default: ''
        placeholder: 'Query',
        
        // updateOnChange: true,
        // updateControlsOnChange: true
    }, 
    {
        type: 'datepicker',
        param: 'datepicker',
        
        innerLabel: 'Date',
        
        // updateOnChange: true,
        // updateControlsOnChange: true
    },
    {
        type: 'range-datepicker',

        paramFrom: 'rangeDatepickerFrom',
        paramTo: 'rangeDatepickerTo',
        
        innerLabel: 'Calendar',

        // updateOnChange: true,
        // updateControlsOnChange: true
    },
    {
        type: 'checkbox',
        param: 'checkbox',
    
        innerLabel: 'Verify',

        // updateOnChange: true,
        // updateControlsOnChange: true
    },
    {
        type: 'button',
        
        innerLabel: 'Go',
        
        theme: 'action',
        updateOnChange: true,
    }
];
```

**Comment by Marginy605** [comment]
- edit chart link (an dash and preview) not working (for instance, edit wizard chart on dash open wizard page by edit link click)
- enable editor selector add on dash
- hide docs tabs ?
- enable for d3 chart
- change to brand color: editro tabs, points in history
- translations in selected chart type

**Comment by korvin89** [comment]
@Marginy605 Hey!

_What I've done:_
- change control template
- fix chart editing from dash/preview
- enable editor selector add on dash
- hide docs tabs
- fix brand colours
- fix translations for selected chart type (only OS)

_What I have not done:_
- translations for all selected chart types #762
- d3 support #764

**Review by melikhov-dev — APPROVED** [review]

**Review by Marginy605 — APPROVED** [review]

**Comment by korvin89** [comment]
Related https://github.com/datalens-tech/datalens/issues/94

**Pull request merged**